README.md

```diff
@@ -10,6 +10,13 @@ With MariaDB:
     % brew tap mroonga/mroonga
     % brew install mroonga --use-homebrew-mariadb
 
+If you have an old mysql, please switch to the latest version:
+
+    % mysql.server stop
```

Comment:
Could you use `brew services stop mysql` here?
`brew services` sub command is more modern way to stop macOS services than this one.